tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -145,8 +146,11 @@ public bool CheckForSecrets(GitAssetsConfiguration assetsConfiguration, string[]
             {
                 if (CheckForSecrets(config, pendingChanges))
                 {
-                    Environment.ExitCode = -1;
-                    return;
+                    if (!ignoreSecretProtection)
+                    {
+                        Environment.ExitCode = -1;
```

Comment:
Is this a not implemented type thing right now?

Reply:
I don't get what you're implying here.